ner_v2/detectors/pattern/phone_number/phone_number_detection.py

```diff
@@ -101,7 +101,19 @@ def detect_entity(self, text, **kwargs):
         """
         self.text = " " + text.lower().strip() + " "
         self.phone, self.original_phone_text = [], []
+
         for match in phonenumbers.PhoneNumberMatcher(self.text, self.country_code, leniency=0):
+            try:
+                national_number_len = len(str(match.number.national_number))
+
+                # Get the national number and check its length is below 8 (including contry code) and \
```

Comment:
This is not making sense. The number returned here doesn't include the country code per the screenshot shared with me, so this will calculate length with country code excluded.

Concretely this will reject +91-9650992 as too short to be a phone number. Please make sure you want that. Also please add some sort of unit test that lets us know if what you want here is working.

![Screenshot 2024-10-07 at 2 40 08 PM](https://github.com/user-attachments/assets/c5912d93-1eb2-44df-af08-c57c1dfc6d5f)

Reply:
@rootAvish - We are doing testing on prestaging once then will move to staging, will that work?